internal/cmddestroy/cmddestroy.go

```diff
@@ -152,10 +153,10 @@ func (r *Runner) runE(c *cobra.Command, args []string) error {
 		}
 	}
 
-	return r.destroyRunner(r, invInfo, dryRunStrategy)
+	return r.destroyRunner(r, invInfo, inv, dryRunStrategy)
 }
 
-func runDestroy(r *Runner, inv inventory.Info, dryRunStrategy common.DryRunStrategy) error {
+func runDestroy(r *Runner, inv inventory.Info, kptInv kptfilev1.Inventory, dryRunStrategy common.DryRunStrategy) error {
```

Comment:
Are there unit tests for this that can be extended for this to show it still works without regression?

Reply:
The existing unit tests should already be capturing this.